hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/viewfs/ViewFileSystem.java

```diff
@@ -118,38 +118,35 @@ protected FsGetter fsGetter() {
    * Caching children filesystems. HADOOP-15565.
    */
   static class InnerCache {
-    private Map<Key, FileSystem> map = new HashMap<>();
+    private ConcurrentHashMap<Key, FileSystem> map = new ConcurrentHashMap<>();
```

Comment:
Minor nit: the interface type ConcurrentMap is slightly preferred in the declaration over the concrete implementation type ConcurrentHashMap. Just a better idiomatic style.

Reply:
Fixed. thanks,